doc/src/devdocs/reflection.md

```diff
@@ -122,11 +122,11 @@ calls and expand argument types automatically:
 ```julia-repl
 julia> @code_llvm +(1,1)
 
-; Function Attrs: sspreq
-define i64 @"julia_+_130862"(i64, i64) #0 {
+;  @ int.jl:53 within `+'
```

Comment:
What's the reason for printing this as `` `+' ``  instead of `` `+` ``? It messes up syntax highlighting sometimes when pasting output.

@vtjnash

Reply:
I'd say just fix it to be paired backticks, which, imo looks better anyway.